dependencies.yaml

```diff
@@ -618,3 +621,11 @@ dependencies:
       - output_types: [conda]
         packages:
           - libwebp=1.3.2 # Required for CVE mitigation: https://nvd.nist.gov/vuln/detail/CVE-2023-4863
+
+  example-dlp:
+    common:
+      - output_types: [conda]
+        packages:
+          - datasets=3.6
+          - onnxruntime=1.22=*cuda
```

Comment:
Do we not want onnx and onnxruntime to be the same version? (1.22 vs 1.15)

Reply:
Surprisingly enough these two don't depend on each other, and appear to be independent.